From 11ed0e98d218dc5c6d5193748c8ee234f0e7dd3a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Felix=20Paul=20K=C3=BChne?= <fkuehne@videolan.org>
Date: Thu, 8 Aug 2013 21:47:46 +0200
Subject: [PATCH] contribs/zvbi: don't reconf after decompressing the source
 code so we don't need to compile gettext just for this

---
 contrib/src/zvbi/rules.mak | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/contrib/src/zvbi/rules.mak b/contrib/src/zvbi/rules.mak
index 7584eed..627ae7c 100644
--- a/contrib/src/zvbi/rules.mak
+++ b/contrib/src/zvbi/rules.mak
@@ -41,7 +41,9 @@ ZVBI_CFLAGS += -DPTW32_STATIC_LIB
 endif
 
 .zvbi: zvbi
+ifndef HAVE_IOS
 	$(RECONF)
+endif
 	cd $< && $(HOSTVARS) CFLAGS="$(ZVBI_CFLAGS)" ./configure $(ZVBICONF)
 	cd $</src && $(MAKE) install
 	cd $< && $(MAKE) SUBDIRS=. install
-- 
1.8.3.1 (Apple Git-46)

